app/src/features/dataset_explorer/DatasetListing.tsx

```diff
@@ -1,82 +1,81 @@
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {props.fields.map((field, index) => (
-          <TableRow key={index}>
-            <TableCell>{field.name}</TableCell>
-            <TableCell>{field.description}</TableCell>
-            <TableCell>{field.data_type}</TableCell>
-            <TableCell>{field.origin_dataset}</TableCell>
-          </TableRow>
-        ))}
-      </TableBody>
-    </Table>
-  );
-}
-
-function DatasetListing(props: {
-  dataset: DatasetMetadata;
-  onPreview: () => void;
-}) {
+function DatasetListing(props: { dataset: DatasetMetadata }) {
   const [expanded, setExpanded] = useState(false);
+
   return (
-    <Paper className={styles.DataSourceListing}>
-      <div className={styles.FlexRow}>
-        <IconButton
-          aria-label="expand dataset"
-          onClick={() => setExpanded(!expanded)}
-          data-testid={"expand-" + props.dataset.id}
-          className={styles.ExpandButton}
+    <Card elevation={3} className={styles.DatasetListing}>
+      <Typography variant="h5" className={styles.DatasetTitle} align="left">
+        {props.dataset.name}
+      </Typography>
+      <Typography className={styles.DataSubtitle} align="left">
+        <a
+          href={props.dataset.data_source_link}
+          target="_blank"
+          rel="noopener noreferrer"
         >
-          {expanded ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
-        </IconButton>
-        <div className={styles.FlexCol}>
-          <div className={styles.FlexRow}>
-            <div className={styles.DatasetTitle}>{props.dataset.name}</div>
-            <Button
-              variant="contained"
-              color="primary"
-              onClick={() => {
-                setExpanded(true);
-                props.onPreview();
-              }}
-              data-testid={"preview-" + props.dataset.id}
-              className={styles.PreviewButton}
-            >
-              Preview
-            </Button>
-          </div>
-          <div>{props.dataset.description}</div>
+          {props.dataset.data_source_name}
+        </a>
+      </Typography>
+      <table className={styles.MetadataTable}>
+        <tbody>
+          <tr>
+            <td>
+              <b>Geographic Level</b>
+            </td>
+            <td>{props.dataset.geographic_level}</td>
+          </tr>
+          <tr>
+            <td>
+              <b>Demographic Granularity</b>
+            </td>
+            <td>{props.dataset.demographic_granularity}</td>
+          </tr>
+          <tr>
+            <td>
+              <b>Update Frequency</b>
+            </td>
+            <td>{props.dataset.update_frequency}</td>
+          </tr>
+          <tr>
+            <td>
+              <b>Latest Update Time</b>
+            </td>
+            <td>{props.dataset.update_time}</td>
+          </tr>
+        </tbody>
+      </table>
+      <AnimateHeight duration={500} height={expanded ? "auto" : 20}>
```

Comment:
Any reason you used this rather than the Collapse component from material ui? Seems like they accomplish the same thing but the Collapse one doesn't add another dependency

Reply:
They wanted the first line of the description to always be visible and Collapse seemed like all or nothing. I thought about programmatically taking the first line of the description out of the Collapse div but that seemed worse than just changing the height of the div.